packages/nimble-components/src/tests/component-status.stories.ts

```diff
@@ -112,9 +112,9 @@ const components = [
         componentName: 'Card',
```

Comment:
Now that you have a storybook page for the component, you can link it here as well.

```suggestion
        componentName: 'Card',
        componentHref: './?path=/docs/incubating-card--docs',
```